pkitesting/src/main/java/io/netty/pkitesting/RevocationServer.java

```diff
@@ -169,7 +169,7 @@ private static byte[] generateCrl(CrlInfo info) {
         CertificateList list = new CertificateList(issuer, now, now, certs.entrySet());
         try {
             Signed signed = new Signed(list.getEncoded(), issuer);
-            return signed.getEncoded();
+            return signed.getEncoded(null);
```

Comment:
We should add a method to include a provider with the issuer cert, and store the provider in the `CrlInfo` to we can pass it here, if any.